common/version.go

```diff
@@ -1,6 +1,6 @@
 package common
 
-const AzcopyVersion = "10.19.0"
+const AzcopyVersion = "10.19.0-preview"
```

Comment:
this should be 10.20.0-preview right? or 10.19.1

Reply:
This patch is over 10.19 on main.